libr/util/p_format.c

```diff
@@ -1075,6 +1075,19 @@ static int r_print_format_struct(RPrint* p, ut64 seek, const ut8* b, int len,
 	return r_print_format_struct_size(fmt, p, mode);
 }
 
+static void* get_args_offset( const char *arg ) {
```

Comment:
Wow, that's a bad mistake! Probably drug or alcohol... Sad story... :D 

I'll fix that ;)